code/modules/mob/living/simple_animal/friendly/mouse.dm

```diff
@@ -90,6 +89,24 @@
 	body_color = "gray"
 	icon_state = "mouse_gray"
 
+/mob/living/simple_animal/mouse/mouse_op
+	name = "mouse operative"
+	minbodytemp = 0
+	maxbodytemp = 5000
+	maxHealth = 20
+	health = 20
+	mob_biotypes = list(MOB_INORGANIC)
+
+/mob/living/simple_animal/mouse/mouse_op/New()
+	..()
+	desc = "Oh no..."
+	icon_state = "mouse_operative"
```

Comment:
why isn't this preset value in class defintion?

Reply:
vg shitcode